From e11621e3e628b49b09b66214a41cf065dfaafc3d Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 27 Nov 2013 17:03:41 +0800
Subject: [PATCH 264/602] ENGR00215174 update to pass build for android 4.4
 UBIFS setting

Enlarge the system partition to 360MB
Reduce cache/data partition since no enough room for system partition

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 sabreauto_6q/ubi/ubinize.ini | 6 +++---
 sabresd_6dq/ubi/ubinize.ini  | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/sabreauto_6q/ubi/ubinize.ini b/sabreauto_6q/ubi/ubinize.ini
index bb663a4..04855c5 100644
--- a/sabreauto_6q/ubi/ubinize.ini
+++ b/sabreauto_6q/ubi/ubinize.ini
@@ -2,7 +2,7 @@
 mode=ubi
 image=./out/target/product/sabreauto_6q/system.img
 vol_id=0
-vol_size=300MiB
+vol_size=360MiB
 vol_type=dynamic
 vol_name=system
 vol_alignment=1
@@ -10,7 +10,7 @@ vol_alignment=1
 [cache]
 mode=ubi
 vol_id=1
-vol_size=200MiB
+vol_size=180MiB
 vol_type=dynamic
 vol_name=cache
 vol_alignment=1
@@ -26,7 +26,7 @@ vol_aligment=1
 [data]
 mode=ubi
 vol_id=3
-vol_size=300MiB
+vol_size=280MiB
 vol_type=dynamic
 vol_name=data
 vol_alignment=1
diff --git a/sabresd_6dq/ubi/ubinize.ini b/sabresd_6dq/ubi/ubinize.ini
index 8b98cbb..d6832a1 100644
--- a/sabresd_6dq/ubi/ubinize.ini
+++ b/sabresd_6dq/ubi/ubinize.ini
@@ -2,7 +2,7 @@
 mode=ubi
 image=./out/target/product/sabresd_6dq/system.img
 vol_id=0
-vol_size=300MiB
+vol_size=360MiB
 vol_type=dynamic
 vol_name=system
 vol_alignment=1
@@ -10,7 +10,7 @@ vol_alignment=1
 [cache]
 mode=ubi
 vol_id=1
-vol_size=200MiB
+vol_size=180MiB
 vol_type=dynamic
 vol_name=cache
 vol_alignment=1
@@ -26,7 +26,7 @@ vol_aligment=1
 [data]
 mode=ubi
 vol_id=3
-vol_size=300MiB
+vol_size=280MiB
 vol_type=dynamic
 vol_name=data
 vol_alignment=1
-- 
1.8.0

